oak/client/policy_metadata.h

```diff
@@ -0,0 +1,27 @@
+#ifndef OAK_CLIENT_POLICY_METADATA_H_
+#define OAK_CLIENT_POLICY_METADATA_H_
+
+#include "include/grpcpp/grpcpp.h"
+
+namespace oak {
+
+// Metadata key used to refer to Oak Policies associated with the gRPC request. This is effectively
+// treated as the name of a custom HTTP header.
+const std::string kOakPolicyMetadataKey = "x-oak-policy";
```

Comment:
I think you need to separate declaration and (single) definition for this.  (Also, ordering of constructors for global objects is not defined, but [C-like types](https://github.com/project-oak/oak/blob/master/oak/common/app_config.h#L26-L30) aren't constructed.)

Reply:
Still not sure I'm doing it right, please let me know if not and I'll fix in a follow-up commit, thanks!

Reply:
absl::string_view should be OK because it has a constexpr constructor from const char*, but it would be more clear to mark this as constexpr.

But why not just use a const char* string literal?

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Reply:
David sent me http://go/totw/140 separately (not visible outside of Google), and that seems to be fine with `string_view`, so I'm going with that.